### PR TITLE
logging disc compilation logs in torchDisc

### PR DESCRIPTION
**Pull request by Yancey1989**
enable logs with
``` bash
PYTORCH_JIT_LOG_LEVEL=">>registry_disc_class:disc_compiler" python mnist.py
```

### CODE REVIEW

File: pytorch_blade/src/ltc/disc_compiler/passes/register_disc_class.cpp [hunk 1]
```diff
@@ -60,12 +62,14 @@ std::tuple<std::string, std::string, std::string> MhloConversaion(
...
 
-std::string CallDiscCompiler(const std::string& mlir_fname) {
+std::tuple<std::string, std::string> CallDiscCompiler(
+    const std::string& mlir_fname) {
   std::string out_fname = mlir_fname + ".out";
   std::string cmd = DiscCMD(mlir_fname, out_fname);
   TORCH_CHECK(
       std::system(cmd.c_str()) == 0, "disc compile failed with cmd: " + cmd);
-  return out_fname;
+  // GRAPH_DEBUG("call disc-compiler-main with cmd: ", cmd);
```
Comment (fortianyou) [inline pytorch_blade/src/ltc/disc_compiler/passes/register_disc_class.cpp:71]: Remove unused

File: pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp [hunk 2]
```diff
@@ -95,10 +95,17 @@ ExecutablePtr CompileToDiscExecutable(
   // Inference shape
   torch::jit::PropagateInputShapes(graph);
   // cluster disc compitable nodes into a sub-graph
+  GRAPH_DEBUG("before DiscFusion\n ", *graph);
   DiscFusion(graph);
+  GRAPH_DEBUG("after DiscFusion, before EliminateDeadCode\n", *graph)
+  // torch::blade::GRAPH_DEBUG("before EliminateDeadCode, after DiscFusion: \n",
```
Comment (fortianyou) [inline pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp:101]: Remove unused codes

File: pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp [hunk 3]
```diff
@@ -11,18 +11,18 @@
...
 #include <ATen/Functions.h>
+#include <torch/csrc/jit/jit_log.h>
 #include <torch/csrc/jit/passes/dead_code_elimination.h>
 #include <torch/csrc/jit/passes/shape_analysis.h>
 #include <torch/csrc/lazy/backend/backend_data.h>
 #include <torch/csrc/lazy/backend/backend_device.h>
 #include <torch/csrc/lazy/ts_backend/ts_backend_impl.h>
 #include <torch/csrc/lazy/ts_backend/ts_lowering_context.h>
 #include <torch/script.h>
+#include "common_utils/utils.h"
```
Comment (qiuxiafei) [inline pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp:23]: Do we have a linter for include order?

**Comment by qiuxiafei** [comment]
What does the `>>` in env var mean?

File: pytorch_blade/src/ltc/disc_compiler/passes/register_disc_class.cpp (hunk 1 above)
Comment (Yancey1989) [reply pytorch_blade/src/ltc/disc_compiler/passes/register_disc_class.cpp]: done.

File: pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp (hunk 2 above)
Comment (Yancey1989) [reply pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp]: done.

File: pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp (hunk 3 above)
Comment (Yancey1989) [reply pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp:23]: Yes, we use clang-format : https://github.com/alibaba/BladeDISC/blob/main/.pre-commit-config.yaml#L12

**Comment by Yancey1989** [comment]
> What does the >> in env var mean?

@qiuxiafei PyTorch JIT logger use `>`, `>>` to represent the log level, ref: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/jit_log.h#L34

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**